nuxeo-runtime/nuxeo-runtime-test/src/main/java/org/nuxeo/runtime/test/runner/Deploy.java

```diff
@@ -36,16 +37,17 @@
  * Example with one module:
  *
  * <pre>
- * @Deploy("org.nuxeo.runtime")
+ * &#64;Deploy("org.nuxeo.runtime")
  * </pre>
  *
  * Example with several modules:
  *
  * <pre>
- * @Deploy({"org.nuxeo.runtime", "org.nuxeo.core:OSGI-INF/component.xml"})
+ * &#64;Deploy({"org.nuxeo.runtime", "org.nuxeo.core:OSGI-INF/component.xml"})
  * </pre>
```

Comment:
If the non UTF-8 character is the single issue I'll edit through github before merging